src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -61,6 +56,7 @@ const useGroupSetting = ({route}) => {
   }, [selectedChannel]);
 
   const submitData = async (withNavigation = true) => {
+    if (!changeImage || changeName) return navigation.goBack();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition in the `submitData` function has been updated to check if there are changes in the image or name before navigating back. This could potentially prevent unnecessary navigation when there are no changes. However, the logic seems incorrect. It should navigate back only when there are no changes in both image and name. The current logic will return even if there's a change in the name but not in the image.
```diff
- if (!changeImage || changeName) return navigation.goBack();
+ if (!changeImage && !changeName) return navigation.goBack();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->